platform/bukkit/src/main/java/me/lokka30/treasury/plugin/bukkit/vendor/BukkitVendor.java

```diff
@@ -43,19 +42,31 @@ public static Platform getPlatformClass() {
         return platform;
     }
 
+    /**
+     * Returns whether we're running any of the specified classes.
+     *
+     * @param classNames the class names to check for
+     * @return any of the specified classes?
+     */
+    private static boolean hasClass(String... classNames) {
+        for (String className : classNames) {
+            try {
+                Class.forName(className);
+                return true;
+            } catch (ClassNotFoundException ignored) {
+            }
+        }
+        return false;
+    }
```

Comment:
I'm being pedantic here - may be worth renaming this method to `hasAnyClasspath`.

Reply:
Doesn't matter, if it works, it works. It's also treasury bukkit plugin internal stuff.